docs/platforms/dotnet/common/configuration/msbuild/project-configuration.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Project Configuration
+description: "Configure additional MSBuild properties in your .NET project."
+sidebar_order: 30
+---
+
+## Project Properties
+
+The following properties can be set via an MSBuild project file (`.csproj`, `.vbproj`, `.fsproj`),
+included in automatically imported project files (`Directory.Build.props`, `Directory.Build.targets`),
+or passed via the .NET CLI (`-p|--property` option of the `dotnet build` command).
+
+<ConfigKey name="SentryImplicitUsings">
+
+C# 10.0 or greater support the language feature _'global using directive'_.
+Depending on the type of .NET project SDK, the MSBuild property [ImplicitUsings](https://learn.microsoft.com/dotnet/core/project-sdk/msbuild-props#implicitusings) automatically adds a set of default `global using` directives to your C# projects.
+When `ImplicitUsings` are enabled, the Sentry SDK also adds commonly used `global using` directives to your C# projects.
+
+If you have namespace conflicts, you can opt out of Sentry's `global using` directives when `ImplicitUsings` are enabled:
+```xml
+<PropertyGroup>
+  <SentryImplicitUsings>false</SentryImplicitUsings>
+</PropertyGroup>
+```
+
+_Available since 4.0.0_
+
+</ConfigKey>
+
+<ConfigKey name="SentryNative">
+
+The Sentry .NET SDK wraps the [Sentry Native SDK](/platforms/native).
+
+In case you run into issues with the native dependencies, you can opt out of including the Sentry Native SDK via:
+```xml
+<PropertyGroup>
+  <SentryNative>false</SentryNative>
+</PropertyGroup>
+```
+or
```

Comment:
@getsentry/docs question: Is there a better way to show alternatives?

Reply:
Yes, but as a user, when would I pick one over the other?

I understand .NET might have the convention of supporting both `false` and `disable` but for the sake of docs, I'd document only one for simplicity sake